src/styles/resume.scss

```diff
@@ -18,7 +18,7 @@
     display: inline-block;
   }
 
-  hr {
+  h2 + hr {
```

Comment:
It seem this css selector is quite broad, it override that `work/item.vue` and `project/item.vue`'s `hr` style.

![image](https://github.com/user-attachments/assets/a30169d9-f338-42f8-99ff-a747ec0c3752)